tests/lib/snaps/test-snapd-daemon-notify/meta/hooks/install

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# stop and disable the service so that the install can continue
+snapctl stop --disable "$SNAP_NAME.notify"
```

Comment:
Isn't it sufficient to disable the service? It won't be started yet anyway, right?

Reply:
No, we need to specifically *disable* the service here in the install hook because if we don't then the installation of the snap will be aborted when the service is started up and it doesn't have permissions to use daemon-notify and exits.

/me remembers that @zyga was the one who first told me about this feature :-)

Reply:
Hmm although if you meant why we are calling stop, it's because there's no `snaptl disable ...`, so we always have to stop and disable it.